src/Aspire.Dashboard/Components/Pages/ConsoleLogs.razor.cs

```diff
@@ -153,7 +153,7 @@ private async ValueTask LoadLogsAsync()
             {
                 var task = _logViewer.SetLogSourceAsync(
                     subscription,
-                    convertTimestampsFromUtc: _selectedResource is ContainerViewModel);
+                    convertTimestampsFromUtc: _selectedResource.ResourceType is "Container");
```

Comment:
ResourceType needs to be string, but I'd like to avoid string comparisons spread throughout code. Could some helper methods be added for comparing known resource types be added? Similar to discussion about known statuses.

Reply:
There is an IsContainer method. Can that be used here?

Reply:
ditto elsewhere in this file

Reply:
Yep, changed to use `IsContainer()`. Searched for all other instances too.